router/transformer/transformer.go

```diff
@@ -361,37 +416,36 @@ func (trans *handle) doProxyRequest(ctx context.Context, proxyReqParams *ProxyRe
 
 	if os.IsTimeout(err) {
 		// A timeout error occurred
-		trans.logger.Errorf(`[TransformerProxy] (Dest-%[1]v) {Job - %[2]v} Client.Do Failure for %[1]v, with %[3]v`, destName, jobID, err.Error())
+		trans.logger.Errorf(`[TransformerProxy] (Dest-%[1]v) Client.Do Failure for %[1]v, with %[2]v`, destName, err.Error())
 		return httpProxyResponse{
 			respData:   []byte{},
 			statusCode: http.StatusGatewayTimeout,
 			err:        err,
 		}
 	} else if err != nil {
 		// This was an error, but not a timeout
-		trans.logger.Errorf(`[TransformerProxy] (Dest-%[1]v) {Job - %[2]v} Client.Do Failure for %[1]v, with %[3]v`, destName, jobID, err.Error())
+		trans.logger.Errorf(`[TransformerProxy] (Dest-%[1]v) Client.Do Failure for %[1]v, with %[2]v`, destName, err.Error())
 		return httpProxyResponse{
 			respData:   []byte{},
 			statusCode: http.StatusInternalServerError,
 			err:        err,
 		}
 	} else if resp.StatusCode == http.StatusNotFound {
 		// Actually Router wouldn't send any destination to proxy unless it already exists
-		// But if accidentally such a request is sent, we'd probably need to handle for better
-		// understanding of the response
+		// But if accidentally such a request is sent, failing instead of aborting
 		notFoundErr := fmt.Errorf(`post "%s" not found`, req.URL)
-		trans.logger.Errorf(`[TransformerProxy] (Dest-%[1]v) {Job - %[2]v} Client.Do Failure for %[1]v, with %[3]v`, destName, jobID, notFoundErr)
+		trans.logger.Errorf(`[TransformerProxy] (Dest-%[1]v) Client.Do Failure for %[1]v, with %[2]v`, destName, notFoundErr)
 		return httpProxyResponse{
 			respData:   []byte{},
-			statusCode: resp.StatusCode,
+			statusCode: http.StatusInternalServerError,
```

Comment:
The chance of wrongly configuring is less. So shouldn't be noisy.

IMO, in case of wrong configuration, it is better to fail with 5xx and retry rather than abort immediately. Gives us time to fix the config.